OpenRA.Game/TextManager.cs

```diff
@@ -0,0 +1,51 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2021 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Primitives;
+using OpenRA.Widgets;
+
+namespace OpenRA
+{
+	public static class TextManager
```

Comment:
Isn't this more like a `ChatManager`? (Operations on the chat, not on text/strings.)

Reply:
I did contemplate on it, but things like `AddSystemLine`  and `Debug` makes that name a bit strange, maybe ChatTextManager?

Reply:
I think `ChatManager` and `ChatTextManager` are not very different. If we want to keep it more general then perhaps something like `TextNotificationsManager`?